src/common/exif.cc

```diff
@@ -733,6 +748,76 @@ static gboolean dt_check_dng_opcodes(Exiv2::ExifData &exifData, dt_image_t *img)
   return has_opcodes;
 }
 
+static gboolean dt_check_lens_correction_data(Exiv2::ExifData &exifData, dt_image_t *img)
+{
+  Exiv2::ExifData::const_iterator pos, posd, posc, posv;
+
+  /*
+   * Sony lens correction data
+   */
+  if(Exiv2::versionNumber() >= EXIV2_MAKE_VERSION(0, 27, 4)
+    && dt_exif_read_exif_tag(exifData, &posd, "Exif.SubImage1.DistortionCorrParams")
+    && dt_exif_read_exif_tag(exifData, &posc, "Exif.SubImage1.ChromaticAberrationCorrParams")
+    && dt_exif_read_exif_tag(exifData, &posv, "Exif.SubImage1.VignettingCorrParams"))
+  {
+    printf("Found sony!\n");
```

Comment:
Remove me!

Reply:
@FreddieWitherden Ok. From everything? (commits, code etc...?)

Reply:
No, just the printf statement!  Must have slipped in when I was doing some debugging.

Reply:
Ah. Ok!